libqtile/config.py

```diff
@@ -457,6 +457,16 @@ def __init__(
         self.height = height if height is not None else 0
         self.previous_group: _Group | None = None
 
+    def __eq__(self, other: object) -> bool:
```

Comment:
Where do we actually test for equivalence?

Reply:
I put the link in the commit message, but this whole function probably deserves a comment noting where, I'll move the discussion to a comment here.